compiler/control/OMROptions.cpp

```diff
@@ -1436,6 +1442,17 @@ OMR::Options::setStatic32BitValue(char *option, void *base, TR::OptionTable *ent
    return option;
    }
 
+char *
+OMR::Options::setStaticPercentage(char *option, void *base, TR::OptionTable *entry)
+   {
+   int32_t percentage = entry->parm2;
+   *((int32_t*)((char*)entry->parm1)) = percentage;
+   if ((percentage < 0) || (percentage > 100))
+      {
+      TR_VerboseLog::writeLineLocked(TR_Vlog_FAILURE,"Percentage must be 0-100 at --> '%s'", option);
```

Comment:
`TR_Vlog_FAILURE` is typically used for failed compilations. In this case writing directly to stderr is probably better, though the message should come at the time we provide the correction to indicate what the threshold has been changed to.